pkg/shell/superuser.jsx

```diff
@@ -88,7 +93,14 @@ class UnlockDialog extends React.Component {
                 </>);
         } else if (state.message) {
             title = _("Administrative access");
-            body = <p>{state.message}</p>;
+            body = (
+                <>
+                    <p>{state.message}</p>
+                    <br />
+                    <Checkbox onChange={this.props.change_remember_access_level_on_unlock}
+                              isChecked={this.props.remember_access_level_on_unlock}
+                              label={ _("Automatically gain administrativ access on next login") } />
```

Comment:
same "administrativ" typo.  However, as this is quite a complex translatable string, could it use the exact same one as above? the difference is marginal, and does not seem too relevant?

Reply:
Yes, they should be the same, sloppy coding.  Thanks for catching!

Reply:
Fixed.